frontend/hub/approvals/hooks/useApprovalFilters.tsx

```diff
@@ -11,24 +11,24 @@ export function useApprovalFilters() {
         label: t('Collection'),
         type: 'string',
         query: 'name',
-        comparison: 'contains',
+        comparison: 'equals',
```

Comment:
Filters in the approval dashboard in old app are using equal match. But maybe it will be better use consistent icontains. This is maybe because old API does not allowed anything than equals.

Reply:
It seems that this API does not allow for partial match. Possible topic for monday meeting :-)

Reply:
oh, then sorry about this :) 

As long as you're keeping in mind to check the filters are the same, :+1: from me :)